assets/build.js

```diff
@@ -1,12 +1,26 @@
 /* eslint-disable no-undef */
 /* eslint-disable no-console */
+/* eslint-disable no-undef */
```

Comment:
`/* eslint-disable no-undef */`  line now is duplicated XD

Reply:
You won't believe this. I had vim open and first line was just truncated LOL

Reply:
You could always use emacs.... XD

Reply:
DUDE